client-v2/src/selectors/configSelectors.ts

```diff
@@ -28,7 +29,9 @@ const capiPreviewURLSelector = createSelector(
 
 const collectionCapSelector = createSelector(
   configSelector,
-  config => (config && config.collectionCap) || Infinity
+  selectIsEditingEditions,
+  (config, isEditingEditions) =>
+    (!isEditingEditions && config && config.collectionCap) || Infinity
```

Comment:
Nice lol

Reply:
Though this will make having a collection cap of zero impossible - which is probably fine :)